addons/mail/static/src/discuss/core/public_web/message_actions.js

```diff
@@ -0,0 +1,12 @@
+import { messageActionsRegistry } from "@mail/core/common/message_actions";
+import { _t } from "@web/core/l10n/translation";
+
+messageActionsRegistry.add("new-sub-channel", {
```

Comment:
`create-thread`, to match with UI

Reply:
I don't really mind but I tried to keep the `sub-channel` wording to avoid confusion with `mail.thread`. Can't we keep the sub-channel part here? maybe `create-sub-channel`?

Reply:
Yeah, the "thread" wording is used a lot technical, while not much exposed to end-user.
Personally, I think "thread" is synonymous to "conversation" (and loosely "chat"), so I like that we use this in the code. Also this is shorter than "conversation" so there's that.

We can keep "thread" in this task for now, as to not make important changes before merge.

_(I'm wondering if some alternative wording could fit better, like "(make/view) topics"? Just a random idea, there might be issues with this naming of the feature... but if there's barely any issue with it, it's unique enough to have no ambiguity with rest of code of Discuss ^^)_